internal/validation/openapi.go

```diff
@@ -78,6 +128,82 @@ func ValidateOpenAPI(ctx context.Context, schemaPath, header, token string, limi
 	return nil
 }
 
+func errorsToTabContents(schema []byte, errs []error) []interactivity.InspectableContent {
+	var contents []interactivity.InspectableContent
+
+	lines := strings.Split(string(schema), "\n")
+
+	for _, err := range errs {
+		vErr := errors.GetValidationErr(err)
+
+		lineNumber := styles.SeverityToStyle(vErr.Severity).Render(fmt.Sprintf("Line %d:", vErr.LineNumber))
+		errType := styles.Dimmed.Render(vErr.Rule)
+		s := fmt.Sprintf("%s %s - %s", lineNumber, errType, vErr.Message)
+
+		content := interactivity.InspectableContent{
+			Summary:      s,
+			DetailedView: getDetailedView(lines, *vErr),
+		}
+
+		contents = append(contents, content)
+	}
+
+	if len(errs) == 0 {
+		s := styles.Emphasized.Render("Congrats, there are no issues!")
+		content := interactivity.InspectableContent{
+			Summary:      s,
+			DetailedView: s,
+		}
+		contents = append(contents, content)
+	}
+
+	return contents
+}
+
+func getDetailedView(lines []string, err errors.ValidationError) string {
+	var sb strings.Builder
+
+	errAndLine := styles.SeverityToStyle(err.Severity).Render(fmt.Sprintf("%s on line %d", err.Severity, err.LineNumber))
+	sb.WriteString(fmt.Sprintf("%s %s\n", errAndLine, styles.Dimmed.Render(err.Rule)))
+	sb.WriteString(err.Message)
+	sb.WriteString("\n\n")
+	sb.WriteString(styles.Emphasized.Render("Surrounding Lines:"))
+	sb.WriteString("\n")
+
+	startLine := err.LineNumber - 5
```

Comment:
if I'm understanding correctly are we rendering the 5 preceding lines and 2 following lines of the error